src/blocks/gallery-stacked/edit.js

```diff
@@ -157,6 +157,8 @@ class GalleryStackedEdit extends Component {
 			lightbox,
 		} = attributes;
 
+		const hasImages = !! images.length;
+
```

Comment:
For posterity: This change was needed because of a race condition. It seems that this script is now processed in a `strict mode` perhaps because the error is related to the order of instantiation. This is perhaps due to changes in the way in which WebPack now compiles files. This is unsubstantiated but likely given the refactors with `wp-scripts`. This was the only identifiable issue related to order of instantiation.